packages/smooth_app/lib/pages/onboarding/country_selector.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
Can this widget be used in the future maybe in things like in the settings or is it made too specialized, if not it probably belongs more in the widgets folder

Reply:
Too specific right now, if needed in settings etc, we can refactor out the stuff that can be reused.